vendor/assets/javascripts/jquery.textile.js

```diff
@@ -46,15 +46,27 @@
     this._defaults = defaults;
     this._name = pluginName;
 
-    this.init();
+    this.init(this.$element.data('textiled'));
   }
 
   Plugin.prototype = {
-    init: function () {
+    init: function ( exists ) {
       // Place initialization logic here
       // You already have access to the DOM element and the options via the instance,
       // e.g., this.element and this.options
-      this._buildContainer();
+      if (exists) {
+        this.options.$help    = $('.textile-help');
```

Comment:
is this code "duplicated" ?

Looks like it is also performed by `_buildContainer`.
So I was thinking if we could do something like:
```
if (!exists) {
  this._buildContainer();
}

...[this code]...
```